package repositories;

import Models.Vehicle;

import java.util.Optional;

public class VehicleRepository {
    public Optional<Vehicle> findVehicleByNumber(String number) {
        return Optional.empty();
    }

    public Vehicle save(Vehicle vehicle) {
        return vehicle;
    }
}
